run_marqo.sh

```diff
@@ -74,8 +74,10 @@ elif [ -z "$VESPA_QUERY_URL" ] && [ -z "$VESPA_DOCUMENT_URL" ] && [ -z "$VESPA_C
     if echo "$RESPONSE" | grep -q '"error-code":"NOT_FOUND"'; then
       echo "Marqo did not find an existing vector store. Setting up vector store..."
 
+      # Generate the application package
+      python3 /app/scripts/vespa_local/vespa_local.py generate-application-package
       # Deploy a dummy application package
-      vespa deploy /app/scripts/vespa_local --wait 300 >/dev/null 2>&1
+      vespa deploy vespa_dummy_application_package --wait 300 >/dev/null 2>&1
```

Comment:
To be uniform, let's use the deploy command from vespa_local as well, since we have it. 
I'd prefer if we just make a new step in `vespa_local` like:
`python3 /app/scripts/vespa_local/vespa_local.py generate-and-deploy-application-package`

Reply:
Having a combined generate and deploy step will also be useful for local users